example.ipynb

```diff
@@ -92,9 +92,54 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Downloading autonomous_greenhouse_regression (size = 887.2 MB): 887226368it [00:33, 26634550.95it/s]                               ouse_regression.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[AgML Download]: Extracting files for autonomous_greenhouse_regression... Done!\n",
+      "\n",
+      "================================================================================\n",
+      "You have just downloaded \u001b[1mautonomous_greenhouse_regression\u001b[0m.\n",
+      "\n",
+      "This dataset is licensed under the \u001b[1mCC BY-SA 4.0\u001b[0m license.\n",
+      "To learn more, visit: https://creativecommons.org/licenses/by-sa/4.0/\n",
+      "\n",
+      "When using this dataset, please cite the following:\n",
+      "\n",
+      "@misc{https://doi.org/10.4121/15023088.v1,\n",
+      "  doi = {10.4121/15023088.V1},\n",
+      "  url = {https://data.4tu.nl/articles/_/15023088/1},\n",
+      "  author = {Hemming,  S. (Silke) and de Zwart,  H.F. (Feije) and Elings,  A. (Anne) and bijlaard,  monique and Marrewijk,  van,  Bart and Petropoulou,  Anna},\n",
+      "  keywords = {Horticultural Crops,  Mechanical Engineering,  FOS: Mechanical engineering,  Artificial Intelligence and Image Processing,  FOS: Computer and information sciences,  Horticultural Production,  FOS: Agriculture,  forestry and fisheries,  Autonomous Greenhouse Challenge,  autonomous greenhouse,  Artificial Intelligence,  image processing,  computer vision,  Horticulture,  Lettuce,  sensors,  non-destructive sensing},\n",
+      "  title = {3rd Autonomous Greenhouse Challenge: Online Challenge Lettuce Images},\n",
+      "  publisher = {4TU.ResearchData},\n",
+      "  year = {2021},\n",
+      "  copyright = {Creative Commons Attribution 4.0 International}\n",
+      "}\n",
+      "\n",
+      "You can find additional information about this dataset at:\n",
+      "https://data.4tu.nl/articles/dataset/3rd_Autonomous_Greenhouse_Challenge_Online_Challenge_Lettuce_Images/15023088/1\n",
+      "\n",
+      "This message will \u001b[1mnot\u001b[0m be automatically shown\n",
+      "again. To view this message again, in an AgMLDataLoader\n",
+      "run `loader.info.citation_summary()`. Otherwise, you\n",
+      "can use `agml.data.source(<name>).citation_summary().`\n",
+      "\n",
+      "You can find your dataset at /data/pvraja/DeformableCNN-PlantTraits/autonomous_greenhouse_regression.\n",
+      "================================================================================\n"
+     ]
+    }
+   ],
    "source": [
     "import agml\n",
     "loader = agml.data.AgMLDataLoader('autonomous_greenhouse_regression', dataset_path = './')"
```

Comment:
Since you're just downloading the dataset and then manually using it, use the following (no need to instantiate a loader):

```
agml.data.download_public_dataset('autonomous_greenhouse_regression', location = '.')
```